Add Windows support (#161)
ak1394/react-native-tts: README.md

import Tts from 'react-native-tts';
```

#### Windows

1. In `windows/myapp.sln` add the `RNTTS` project to your solution:

   - Open the solution in Visual Studio 2019
   - Right-click Solution icon in Solution Explorer > Add > Existing Project
   - Select `node_modules\react-native-tts\windows\RNTTS\RNTTS.vcxproj`

> 2. In `windows/myapp/myapp.vcxproj` ad a reference to `RNTTS` to your main application project. From Visual Studio 2019:

Reviewer: ```suggestion
2. In `windows/myapp/myapp.vcxproj` add a reference to `RNTTS` to your main application project. From Visual Studio 2019:
```

   - Right-click main application project > Add > Reference...
   - Check `RNTTS` from Solution Projects.

3. In `pch.h` add `#include "winrt/RNTTS.h"`.

4. In `app.cpp` add `PackageProviders().Append(winrt::RNTTS::ReactPackageProvider());` before `InitializeComponent();`.

### Windows MediaElement

On Windows, `react-native-tts` needs a media element in the visual tree to output the speech stream.
It will look for a XAML `MediaElement` named `RNTextToSpeech` and use it.
This needs to be added to the XAML tree of the screens where `react-native-tts` is used.

> As an example, in `windows/myapp/MainPage.xaml` from the `react-native-tts` app template this can be done by adding a XAML `Grid` with an invisible `MediaElement` alongside the `ReactRootView`. Change `windows/myapp/MainPage.xaml` from:

Reviewer: ```suggestion
As an example, in the `windows/myapp/MainPage.xaml` file from the `react-native-tts` app template, you can add a XAML `Grid` with an invisible `MediaElement` alongside the `ReactRootView`. Change `windows/myapp/MainPage.xaml` from:
```

```xaml
<Page
  ...
  >
  <react:ReactRootView
    x:Name="ReactRootView"
    ...
  />
</Page>
```
to
```xaml
<Page
  ...
  >
  <Grid>

>     <MediaElement x:Name="RNTextToSpeech" />

Reviewer: MediaElement has been replaced by MediaPlayerElement - [link to MSDN](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.mediaelement?view=winrt-19041#remarks). We discourage use of MediaElement; can you double check that this still works assuming MediaPlayerElement is used instead?
